File: apps/problems/01-matrix/boilerplate-full/function.java

    import java.io.*;
    import java.util.*;

    

    //##USER_CODE_HERE##

    public class Main {
      
    public static void main(String[] args) throws IOException {
      BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
      int m_mat = Integer.parseInt(br.readLine().trim());
      int n_mat = Integer.parseInt(br.readLine().trim());
      List<List<Integer>> mat = new ArrayList<>();
      for (int i = 0; i < m_mat; i++) {
        List<Integer> innerList = new ArrayList<>();
        String[] row = br.readLine().trim().split("\\s+");
        for (int j = 0; j < n_mat; j++) {
          innerList.add(Integer.parseInt(row[j]));
        }
        mat.add(innerList);
      }
      Solution obj = new Solution();
      List<List<Integer>> result = obj.matrix(mat);
      
  for (List<Integer> row : result) {
    for (Integer item : row) {
      System.out.print(item + " ");
    }
    System.out.println();
  }
      br.close();
    }
      
    }
  

File: apps/problems/01-matrix/boilerplate-full/function.cpp
#include <iostream>
#include <vector>
#include <string>
#include <sstream>
#include <climits>



//##USER_CODE_HERE


int main() {
    int m_mat;
std::cin >> m_mat;
int n_mat;
std::cin >> n_mat;
std::vector<std::vector<int>> mat(m_mat, std::vector<int>(n_mat));

for(int i = 0; i < m_mat; ++i) {
    for(int j = 0; j < n_mat; ++j) {
        std::cin >> mat[i][j];
    }
}
    Solution obj;
    std::vector<std::vector<int>> result = obj.matrix(mat);
    
for (const auto& row : result) {
  for (const auto& item : row) {
      std::cout << item << " ";
  }
  std::cout << std::endl;
}
    return 0;
}
